roles/docker/tasks/main.yml

```diff
@@ -36,6 +36,15 @@
   tags:
     - prebake-for-dev
 
+- stat: path=/var/docker_images
+  register: docker_images
```

Comment:
this task can also be tagged with `prebake-for-dev`, that way we don't need the explicit `is defined` check below. I tend to avoid `is defined` checks where possible as it help tasks to fail and be caught easily in scenarios where variable might not be defined due to reasons we may not foresee.